deployment/local/kustomization.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: default
+resources:
+- deployment.yaml
+- statistics-dashboard-ui-service.yaml
+- ingress.yaml
+configMapGenerator:
+- name: statistics-dashboard-ui-environment
```

Comment:
This should be better created as a secret(similarly to configmaps) and mounted as such(on all environments), for better security.

Reply:
Done